### PR TITLE
Update creating-modules.md

### PR DESCRIPTION
**Pull request by Wunderbaeumchen99817**
**Contribution description**
Adds example based documentation about adding glue code for existing modules

**Issues/PRs references**
N/A

### CODE REVIEW

File: doc/doxygen/src/creating-modules.md [hunk 1]
```diff
@@ -163,6 +163,27 @@ See `sys/ztimer/Makefile` for an example in code.
 `SUBMODULES` can also be true-pseudomodules, or become one by conditionally excluding
 the source files by adding them to `SUBMODULES_NO_SRC`.
 
+## creating glue-code for existing modules
+
+After creating a new pseudomodule, make sure to add your glue functions. The perfect location would be inside the base-module your glueing to.
```
Comment (aabadie) [inline doc/doxygen/src/creating-modules.md:168]:
```suggestion
After creating a new pseudomodule, make sure to add your glue functions. The perfect location would be inside the base-module you are glueing to. 
```

**Comment by mguetschow** [comment]
Thanks for the write-up! Since this mostly concerns the PSA Crypto API, I would rather expect as part of its documentation.

Actually, there is already a porting guide for both software and hardware backends present: https://doc.riot-os.org/group__sys__psa__crypto.html#porting-guide
Would you mind comparing that information to your findings and potentially enhancing that one instead? You can find the source code in https://github.com/RIOT-OS/RIOT/blob/master/sys/psa_crypto/doc.txt

**Comment by mguetschow** [comment]
Thanks for the changes. However, there is already some documentation on adding glue code in that same file. Could you maybe compare the existing documentation with your workflow and make changes where necessary instead of adding a new section?

(Also I think you might have wanted to edit the PR description of another PR of yours instead :P)

**Comment by mguetschow** [comment]
@Wunderbaeumchen99817 would you mind updating this PR so we can have it in the next release?

**Comment by mguetschow** [comment]
Closing this as stale.

**Pull request closed without merging**